test/evergreen/code_coverage/per_test_code_coverage.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import argparse
+import concurrent.futures
+import json
+import logging
+import os
+import shutil
+import subprocess
+import sys
+from datetime import datetime
+from pathlib import Path
+from code_coverage_utils import check_build_dirs, run_task_lists_in_parallel
+
+
+def get_logger(verbose: bool) -> logging.Logger:
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(level=logging.DEBUG if verbose else logging.INFO)
```

Comment:
I believe we don't need a function for this usage. Basically after `logging.basicConfig(level=logging.DEBUG if verbose else logging.INFO)` is called, you can directly call `logging.info`, `logging.warning` and `logging.error `for your appropriate message.

The setting of the basicConfig function also means that you can use the logging module across different files as long as you import it and the verbosity level still sticks. Therefore I would recommend doing it for code_change_helpers.py, code_change_info.py, code_change_repro.py, code_coverage_utils.py, and parallel_code_coverage.py.

Reply:
That's a good idea, and I've make the changes.